addons/interaction/XEH_preStart.sqf

```diff
@@ -1,3 +1,25 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+if !hasInterface exitWith {};
+
+private _replaceTerrainClasses = QUOTE( \
+    getNumber (_x >> 'scope') == 2 \
+    && {getNumber (_x >> QQGVAR(replaceTerrainObject)) > 0} \
+) configClasses (configFile >> "CfgVehicles");
```

Comment:
SQF Validator thinks it's missing a semi-colon somewhere here. :thinking:

Reply:
already reported in #5979